config.md

```diff
@@ -289,14 +290,16 @@ For Windows based systems the user structure has the following fields:
 
 ## Platform-specific configuration
 
-[**`platform.os`**](#platform) is used to lookup further platform-specific configuration.
+[**`platform.os`**](#platform) is used to specify platform-specific configuration.
+Runtime implementations MAY support any valid values for platform-specific fields as part of this configuration.
+Implementations MUST generate a clear error message for valid values it chooses to not support and MUST generate an error when invalid values are encountered.
```

Comment:
does "a clear error message" mean that the implementation would not continue running and would exit non-zero? Just curious.